src/app/static/src/app/apiService.ts

```diff
@@ -116,13 +127,17 @@ export class APIService<S extends string, E extends string> implements API<S, E>
         if (!failure || !failure.status) {
             throw new Error("Could not parse API response");
         }
-        if (this._onError) {
+        else if (this._onError) {
             this._onError(failure);
         } else {
-            throw new Error(APIService.getFirstErrorFromFailure(failure));
+            this._addErrorToStore(APIService.getFirstErrorFromFailure(failure));
```

Comment:
I'm including a commit to errors here so we can see the module in action - any remaining unhandled throws will be added to the module in mrc-719. You can force this error condition by killing the hintr container then attempting to upload a file. The files coming out of the backend are pretty useless at the moment, but should include an error code to quote once https://github.com/mrc-ide/hint/pull/168 is merged.

Reply:
We could also include the support email address in the header message when the backend PR is merged.